src/s3ql/umount.py

```diff
@@ -89,14 +89,27 @@ def get_cmdline(pid):
     cannot be determined for other reasons, log warning
     and return None.
     '''
+    
+    output = None
+    if os.path.isdir('/proc'):
+        cmd_path = '/proc/%d/cmdline' % pid
+        if os.path.isfile(cmd_path):
+            try:
+                with open(cmd_path, 'r') as cmd_file:
+                    output = cmd_file.read()
+
+            except Exception:
```

Comment:
No catch-all exception handlers please. I believe what you want to catch is only FileNotFound, which actually makes the `isfile` (or `exists`) check superfluous.

Reply:
I see. Changed.